gridstatus/miso.py

```diff
@@ -93,18 +91,32 @@ def get_fuel_mix(
         """Get the fuel mix for a given day for a provided MISO.
 
         Arguments:
-            date (datetime.date, str): "latest", "today", or an object
+            date (datetime.date, str): "latest", "today", "yesterday", or an object
                 that can be parsed as a datetime for the day to return data.
 
             verbose (bool, optional): print verbose output. Defaults to False.
 
         Returns:
             pandas.DataFrame: DataFrame with columns "Time", "Load", "Fuel Mix"
         """
-        if date != "latest":
-            raise NotSupported()
+        if (
+            date == "latest"
+            or date == "today"
+            or utils.is_today(
+                date,
+                tz=self.default_timezone,
+            )
+        ):
+            # https://public-api.misoenergy.org/api/FuelMix/Today is currently not
+            # returning any data.
+            url = "https://public-api.misoenergy.org/api/FuelMix"
+        elif utils.is_yesterday(date, tz=self.default_timezone):
+            url = "https://public-api.misoenergy.org/api/FuelMix/Yesterday"
```

Comment:
I just queried this yesterday file (10:25 pm EST on 12/10) and got back data for today....

<img width="1050" height="2760" alt="Image" src="https://github.com/user-attachments/assets/e291dbba-de68-40c2-913f-1e5bb5f8bd8b" />

[Yesterday.json](https://github.com/user-attachments/files/24093326/Yesterday.json)

Reply:
Hmm. Right now, both today and yesterday are working as expected.